package.json

```diff
@@ -1,18 +1,10 @@
 {
-  "name": "vanilla-timely-plugin",
+  "name": "vanilla-timely",
   "version": "1.0.0",
-  "description": "Load Vanilla JS Timely Modal plugin from any site",
+  "description": "Load vanilla JS Timely Modal from any site",
```

Comment:
Load Ditto Insurance's timely slot booking application in a modal from any website